### ISSUE TITLE
Multiple namespaces

### ISSUE THREAD

**Issue by heynikhil** [socketio/socket.io-admin-ui#6]
We have an option like give namespace,

```
instrument(global.io, {
    auth: false,
    namespaceName: '/',
});
```

What if we have dynamic namespaces?

**Comment by darrachequesne** [comment]
Arf, that's a good remark.

Dynamic namespaces are currently not supported, since we register event listeners only once when calling `instrument()`: https://github.com/socketio/socket.io-admin-ui/blob/ad0ec0de08139e0c88a8b432fe94d3822363580a/lib/index.ts#L389

Two possible solutions:
- have a list of registered namespaces, and periodically check if there are new namespaces
- make the server emit an event when a namespace is created

**Comment by darrachequesne** [comment]
This should be fixed by https://github.com/socketio/socket.io-admin-ui/commit/74f1c20f6ad878c3d11c5fc80dd8d12ee02d7bfb, included in version `0.1.2`.
